### PR TITLE
loading and saving models with dill package

### PR DESCRIPTION
**Pull request by aprotopopov**
I'm not sure about workaround for saving and deleting permissions, so test could failed for that reason. Needed [dill](https://github.com/uqfoundation/dill) package for saving, loading and for test.

### CODE REVIEW

File: lifetimes/estimation.py [hunk 1]
```diff
@@ -31,6 +31,16 @@ def _unload_params(self, *args):
             raise ValueError("Model has not been fit yet. Please call the .fit method first.")
         return [self.params_[x] for x in args]
 
+    def save_model(self, path):
+        import dill
```
Comment (CamDavidsonPilon) [inline lifetimes/estimation.py]: What are your thoughts on adding this to the `reqiurements.txt`?

Comment (aprotopopov) [inline lifetimes/estimation.py]: ok, then `import dill` could be placed on top of the file.

**Comment by aprotopopov** [comment]
Changed requirements to include `dill` package by default

**Comment by CamDavidsonPilon** [comment]
For tests to pass, we also need to update `.tavis.yml` line 55

**Comment by aprotopopov** [comment]
Yeah, all checks passed for the first time.

**Comment by CamDavidsonPilon** [comment]
Thanks for the contribution!

**Pull request merged**